src/test/java/mat/server/FeatureFlagServiceImplTest.java

```diff
@@ -0,0 +1,46 @@
+package mat.server;
+
+import mat.dao.impl.FeatureFlagDAOImpl;
+import mat.model.FeatureFlag;
+import org.junit.Before;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.times;
+
+@ExtendWith(MockitoExtension.class)
+class FeatureFlagServiceImplTest {
+
+    @Mock
+    FeatureFlagDAOImpl featureFlagDAO;
+
+    @InjectMocks
+    FeatureFlagServiceImpl featureFlagServiceImpl;
+
+    @Before
+    public void init() {
+        MockitoAnnotations.initMocks(this);
```

Comment:
this is not needed, since you use MockitoExtension

Reply:
Initialization is removed